### PR TITLE
Input: update stories

### PR DESCRIPTION
**Pull request by spmonahan**
## Current Behavior

1. Input stories use _Title Case for Labels_.
2. The content before/after stories are not copy-pasteable from the Storybook code view

## New Behavior

1. Input stories use _Sentence case for labels_.
2. The content before/after stories are copy-pasteable from the Storybook code view

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `187.182 kB`<br />`51.609 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `31.883 kB`<br />`10.516 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `22.861 kB`<br />`7.512 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `43.378 kB`<br />`12.277 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/766edd211ca8e3ba723f07b7bf10284081df406b'>766edd211ca8e3ba723f07b7bf10284081df406b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/Avatar-mount_base.html">1416</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/Avatar-mount.html">1348</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/Button-mount_base.html">1029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/Button-mount.html">1047</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProvider-mount_base.html">1911</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProvider-mount.html">1906</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProviderWithTheme-mount_base.html">784</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProviderWithTheme-mount.html">737</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">712</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">723</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">780</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">745</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/MakeStyles-mount_base.html">1977</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23724/perf-test-react-components/MakeStyles-mount.html">1974</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by ecraig12345 — APPROVED** [review]
Thanks! 🎉

File: packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx [hunk 1]
```diff
@@ -41,22 +41,22 @@ export const Appearance = () => {
...
     <div className={styles.base}>
       <div className={styles.field}>
-        <Label htmlFor={outlineId}>Outline Input Appearance (default)</Label>
+        <Label htmlFor={outlineId}>Outline appearance (default)</Label>
         <Input appearance="outline" id={outlineId} />
       </div>
 
       <div className={styles.field}>
-        <Label htmlFor={underlineId}>Underline Input Appearance </Label>
+        <Label htmlFor={underlineId}>Underline appearance </Label>
```
Comment (sopranopillow) [inline packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx:49]: nit:

```suggestion
        <Label htmlFor={underlineId}>Underline appearance</Label>
```

File: packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx [hunk 2]
```diff
@@ -41,22 +41,22 @@ export const Appearance = () => {
...
         <Input appearance="underline" id={underlineId} />
       </div>
 
       <div className={mergeClasses(styles.field, styles.filledLighter)}>
-        <Label htmlFor={filledLighterId}>Filled Lighter Input Appearance </Label>
+        <Label htmlFor={filledLighterId}>Filled lighter appearance </Label>
```
Comment (sopranopillow) [inline packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx:54]:
```suggestion
        <Label htmlFor={filledLighterId}>Filled lighter appearance</Label>
```

File: packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx [hunk 3]
```diff
@@ -41,22 +41,22 @@ export const Appearance = () => {
...
         <Input appearance="filled-lighter" id={filledLighterId} />
       </div>
 
       <div className={mergeClasses(styles.field, styles.filledDarker)}>
-        <Label htmlFor={filledDarkerId}>Filled Darker Input Appearance </Label>
+        <Label htmlFor={filledDarkerId}>Filled darker appearance </Label>
```
Comment (sopranopillow) [inline packages/react-components/react-input/src/stories/Input/InputAppearance.stories.tsx:59]:
```suggestion
        <Label htmlFor={filledDarkerId}>Filled darker appearance</Label>
```

**Review by sopranopillow — APPROVED** [review]
lgtm!

**Review by khmakoto — APPROVED** [review]

**Pull request closed without merging**